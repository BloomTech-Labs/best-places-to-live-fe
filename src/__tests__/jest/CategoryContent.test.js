import React from "react";
import CategoryContent from "../../CategoryContentHelper";
import renderer from "react-test-renderer";

describe("<CategoryContent />", () => {
  it("Matches Snapshot", () => {
    const tree = renderer.create(<CategoryContent />).toJSON();
    // @ts-ignore
    expect(tree).toMatchSnapshot();
  });
});
