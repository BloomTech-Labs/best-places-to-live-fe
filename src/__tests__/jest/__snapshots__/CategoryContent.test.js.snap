// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CategoryContent /> Matches Snapshot 1`] = `
<div
  className="sc-bdVaJa fKJtXt"
>
  <div
    className="sc-bwzfXH jKmqkc"
  >
    <h1>
      How we can help a Homie out
    </h1>
    <p>
      With over 5,000 cities available to explore our categories can help you easily find a place to call home. Use them to find the top scoring cities in the category that's most significant to you.
    </p>
  </div>
  <div
    className="sc-htpNat dVNZcv"
  >
    <div
      className="sc-bxivhb icHQkK"
    >
      <img
        className="sc-ifAKCX dAisZH"
        src="budget.png"
      />
      <h4>
        Cost of Living
      </h4>
      <p>
        The amount of money needed to sustain a certain standard of living by affording basic expenses such as housing, food, taxes, and healthcare.
      </p>
    </div>
    <div
      className="sc-bxivhb icHQkK"
    >
      <img
        className="sc-ifAKCX dAisZH"
        src="travel.png"
      />
      <h4>
        Commute
      </h4>
      <p>
        A regular journey of some distance such as to and from your place of work. How long will it take the homies to get where they want to go?
      </p>
    </div>
    <div
      className="sc-bxivhb icHQkK"
    >
      <img
        className="sc-ifAKCX dAisZH"
        src="police-car.png"
      />
      <h4>
        Safety
      </h4>
      <p>
        A collection of overall crime rates in the area so all our homies can stay safe!
      </p>
    </div>
    <div
      className="sc-bxivhb icHQkK"
    >
      <img
        className="sc-ifAKCX dAisZH"
        src="tolerance.png"
      />
      <h4>
        Tolerance
      </h4>
      <p>
        We respect homies of all races, religions, and orientations. Find a place to call home that does too!
      </p>
    </div>
  </div>
</div>
`;
